debian/control

```diff
@@ -0,0 +1,13 @@
+Source: python-ev3dev
+Maintainer: Denis Demidov <dennis.demidov@gmail.com>
+Section: python
+Priority: optional
+Build-Depends: python-setuptools (>= 0.6b3), python-all-dev (>= 2.6.6-3), debhelper (>= 7)
```

Comment:
If you change the source package version above, then the `debhelper` version here gets changed to match.